Loop/Managers/DeviceDataManager.swift

```diff
@@ -1348,6 +1348,15 @@ extension DeviceDataManager: LoopDataManagerDelegate {
             self.crashRecoveryManager.dosingFinished()
         }
     }
```

Comment:
REMOTE 2.0: WILL REMOVE PRE-MERGE

We use this protocol method to trigger when to fetch new commands from our plugins. I assume this will run about every 5 minutes.

Reply:
There is already a `.loopFinished` notification, you could use to trigger updates based on Loop completion.

Reply:
done

Reply:
This updated was added but since removed after I got rid of all Remote 2.0 pieces.